src/index.js

```diff
@@ -13,7 +14,9 @@ ReactDOM.render(
   <React.StrictMode>
     <BrowserRouter>
       <ProductProvider>
-        <App />
+        <WhishlistProvider>
```

Comment:
Take care of typos, here it would be `Wishlist`

Reply:
updated. ✅